### ISSUE TITLE
Feature request: Option to auto-populate orgmode notes file

### ISSUE THREAD

**Issue by WorldsEndless** [tmalsburg/helm-bibtex#46]
First off, helm-bibtex is AWESOME. I'm not sure I've yet come down from my high from learning about this to help with my research. 

I use the orgmode notes option a great deal. I find I start all my note files the same way: by converting key parts of the bibtex entry into org tags. For example, starting my notes file: 

    #+TITLE: [[http://books.google.com/books?id%3Dy8GiAwAAQBAJ][Event Cognition]] (2014)
    #+AUTHOR: Radvansky, G.A. and Zacks, J.M.

from: 

    @book{radvansky2014event,
      publisher = {Oxford University Press},
      year = {2014},
      url = {http://books.google.com/books?id=y8GiAwAAQBAJ},
      isbn = {9780199898145},
      author = {Radvansky, G.A. and Zacks, J.M.},
      title = {Event Cognition},
    }

It would be nice if there were a function/variable to do this automatically if the notes are not yet created, since I'm visiting them via selection of a bibtex entry anyway.

**Comment by tmalsburg** [comment]
Glad to hear that you like helm-bibtex.  There is another issue related to note-taking, #40, and the solution will probably involve some kind of user-defined template for new entries in the notes file.  Stay tuned.

**Comment by WorldsEndless** [comment]
Very nice! Come to think of it, I'd probably switch to one-file notes, too, in order to get easier searching capabilities and the export options.

**Comment by tmalsburg** [comment]
Yeah, the one file solution has some benefits and I will probably switch, too, once the code is in place.

**Comment by tmalsburg** [comment]
I addressed your feature request in the branch note-files.  The template defined in `helm-bibtex-notes-template` is now used to populate new note files.  I'll test this branch and will merge it into master if it works ok.

**Comment by tmalsburg** [comment]
As of 5d028b983465e997cc59dd0237a171eab8fb56ba, this feature is now part of the master branch.  You can customize either `helm-bibtex-notes-template-many-files` (in case that you have one notes file per entry) or `helm-bibtex-notes-template-one-file` (if all notes are stored in one file).

**Comment by tmalsburg** [comment]
Here is [code](https://github.com/tmalsburg/helm-bibtex#convert-multiple-note-files-to-one-notes-file) for converting multiple note files to one note file.
